docs/podman-generate-systemd.1.md

```diff
@@ -27,35 +30,74 @@ Set the systemd restart policy.  The restart-policy must be one of: "no", "on-su
 "on-watchdog", "on-abort", or "always".  The default policy is *on-failure*.
 
 ## Examples
-Create a systemd unit file for a container running nginx:
 
+Create a systemd unit file for a container running nginx with an *always* restart policy and 1-second timeout on stdout.
 ```
-$ sudo podman generate systemd nginx
+$ podman generate systemd --restart-policy=always -t 1 nginx
+# container-c21da63c4783be2ac2cd3487ef8d2ec15ee2a28f63dd8f145e3b05607f31cffc.service
+
 [Unit]
 Description=c21da63c4783be2ac2cd3487ef8d2ec15ee2a28f63dd8f145e3b05607f31cffc Podman Container
+
 [Service]
-Restart=on-failure
+Restart=always
 ExecStart=/usr/bin/podman start c21da63c4783be2ac2cd3487ef8d2ec15ee2a28f63dd8f145e3b05607f31cffc
-ExecStop=/usr/bin/podman stop -t 10 c21da63c4783be2ac2cd3487ef8d2ec15ee2a28f63dd8f145e3b05607f31cffc
+ExecStop=/usr/bin/podman stop -t 1 c21da63c4783be2ac2cd3487ef8d2ec15ee2a28f63dd8f145e3b05607f31cffc
 KillMode=none
 Type=forking
 PIDFile=/var/run/containers/storage/overlay-containers/c21da63c4783be2ac2cd3487ef8d2ec15ee2a28f63dd8f145e3b05607f31cffc/userdata/conmon.pid
+
 [Install]
 WantedBy=multi-user.target
 ```
 
-Create a systemd unit file for a container running nginx with an *always* restart policy and 1-second timeout.
+Create systemd unit files for a pod with a simple alpine container.
 ```
-$ sudo podman generate systemd --restart-policy=always -t 1 nginx
+$ podman pod create --name systemd-pod
+$ podman create --pod systemd-pod alpine top
+$ podman generate systemd --name systemd-pod
```

Comment:
So the command used here and below (the STDOUT one) are identical - missing the --files flag here?

Reply:
Good catch! Fixed.